themes/default/content/docs/clouds/aws/guides/iam.md

```diff
@@ -104,66 +84,10 @@ An AWS managed policy is a standalone policy that is created and administered by
 the policy has its own Amazon Resource Name (ARN) that includes the policy name. For example,
 `arn:aws:iam::aws:policy/IAMReadOnlyAccess` is an AWS managed policy.
 
-In places that accept a policy ARN, such as the `RolePolicyAttachment` resource, you can pass the ARN as a string,
-but that requires that you either memorize or look up the ARN each time. Instead, you can use the strongly typed
-`ManagedPolicy` enum, which exports a collection of constants for all available managed policies.
-
-For example, instead of typing out the ARN by hand, we can just reference `ManagedPolicy`'s `IAMReadOnlyAccess`
+In places that accept a policy ARN, such as the `RolePolicyAttachment` resource, you can pass the ARN as a string, but that requires that you either memorize or look up the ARN each time. Instead, you can use the strongly typed `ManagedPolicy` enum, which exports a collection of constants for all available managed policies. For example, instead of typing out the ARN by hand, we can just reference `ManagedPolicy`'s `IAMReadOnlyAccess`
```

Comment:
```suggestion
In places that accept a policy ARN, such as the `RolePolicyAttachment` resource, you can pass the ARN as a string, but that requires that you either memorize or look up the ARN each time. Instead, you can use the strongly typed `ManagedPolicy` enum, which exports a collection of constants for all available managed policies. For example, instead of typing out the ARN by hand, you can just reference `ManagedPolicy`'s `IAMReadOnlyAccess`
```

There are several other instances of this in the doc that we should clean up.